lib/worker.js

```diff
@@ -18,6 +18,14 @@ const data = Buffer.from(dataBuf)
 async function start () {
   let fn
   try {
+    if(typeof process !== "undefined"){
```

Comment:
Please apply the linting standard of the project. `npm test` will report an error here.